engine/execution/computation/computer/computer.go

```diff
@@ -239,11 +239,19 @@ func (e *blockComputer) executeSystemCollection(
 		return txIndex, fmt.Errorf("could not get system chunk transaction: %w", err)
 	}
 
-	err = e.executeTransaction(tx, colSpan, collectionView, programs, systemChunkCtx, collectionIndex, txIndex, res)
+	err, txerr := e.executeTransaction(tx, colSpan, collectionView, programs, systemChunkCtx, collectionIndex, txIndex, res)
 	txIndex++
 	if err != nil {
+		return txIndex, err
+	} else if txerr != nil {
+		e.log.Err(txerr).
+			Hex("block_id", logging.Entity(systemChunkCtx.BlockHeader)).
+			Str("system_chunk_error", "true").
```

Comment:
We might use something more generic like "critical_error": "true", so we can add it to other places as well.